.eslintrc

```diff
@@ -67,9 +67,11 @@
         "@typescript-eslint/ban-ts-ignore": "off",
         "@typescript-eslint/explicit-function-return-type": "off",
         "@typescript-eslint/no-explicit-any": "off",
+        "@typescript-eslint/no-non-null-assertion": "off",
```

Comment:
@P0lip Are we sure about silencing this globally (rather than on demand, when a precise line needs it)? Do we have that much issue with this (in which case that might put under the light some type design issue)?

Reply:
It's just for test files, not source files. I believe it's fair to use some assertions in tests, since you tend to provide some crappy data there if you test error handling, for instance. We could obviously do it on a case basis, but hm, up to you. I'm fine with both.